app/helpers/dashboard_helper.rb

```diff
@@ -15,6 +15,8 @@ def dashboard_actions
 
   def render_widget(widget)
     render(:partial => widget.template, :locals => widget.data)
+  rescue ActionView::MissingTemplate
+    _("Missing template '%{template}' for widget '%{widget}'. The widget or a plugin providing it was probably removed") % { :widget => widget.name, :template => widget.template }
```

Comment:
inecas in outdated comment above:
> I don't want to reraise: the point of this bug was that, if you uninstall the plugin, you start getting 500 in dashboard, and there was not easy way how to fix that. With this fix, you get mostly empty widget space, with the message why it's that. Reraising the error would take us to the start

I agree that we shouldn't raise any exception here.